server/chat/backend/agent/orchestrator/role_registry.py

```
@@ -0,0 +1,100 @@
+                self._roles[role.name] = role
+                logger.info("RoleRegistry: loaded role %r", role.name)
+            except Exception:
+                logger.exception("RoleRegistry: failed to load %s", md_file.name)
+
+    def list_all(self) -> list[RoleMeta]:
+        return sorted(self._roles.values(), key=lambda r: r.rca_priority)
+
+    def get(self, name: str) -> Optional[RoleMeta]:
+        return self._roles.get(name)
+
+    def list_available_roles(self, user_id: str) -> list[RoleMeta]:
```

Ruff (0.15.12)

[warning] Unused method argument: `user_id`

(ARG002)